inference-engine/src/vpu/graph_transformer/src/middleend/passes/adjust_data_location.cpp

```diff
@@ -234,7 +234,7 @@ void PassImpl::adjustModelForMemReqs(const Model& model) {
         VPU_THROW_UNLESS(!failedData || failedData->memReqs() == MemoryType::CMX,
             R"(Stage "{}" of type "{}" requested {} bytes in {} for output "{}", while there is only {} bytes is free)",
             failedStage->name(), failedStage->type(), calcAllocationSize(failedData), failedData->memReqs(), failedData->name(),
-                         allocator.freeMemoryAmount(failedData->memReqs()));
+                        failedData->memReqs() == MemoryType::CMX ? allocator.freeCMXMemoryAmount() : -1);
```

Comment:
`failedData->memReqs() == MemoryType::CMX` is always false in error message arguments. Take a look at the condition of assertion.
I'd suggest something like this:
```
VPU_THROW_UNLESS(!failedData || failedData->memReqs() == MemoryType::CMX,
R"(Request {} bytes in {} for output "{}" failed for stage "{}" of type "{}"",
calcAllocationSize(failedData), failedData->memReqs(), failedData->name(), failedStage->name(), failedStage->type());
```